rollup.config.js

```diff
@@ -9,34 +10,35 @@ import license from 'rollup-plugin-license';
 
 import {version} from './packages/node_modules/ciscospark/package.json';
 
-export default {
-  input: `${__dirname}/packages/node_modules/ciscospark/browser.js`,
-  output: {
-    name: 'CiscoSpark',
-    file: `${__dirname}/packages/node_modules/ciscospark/umd/ciscospark.min.js`,
-    format: 'iife', // since this is for the browser only use IIFE instead of UMD
-    sourceMap: true
-  },
-  plugins: [
-    cleaner({
-      targets: [
-        `${__dirname}/packages/node_modules/ciscospark/umd/`
-      ]
-    }),
-    json({
-      include: ['packages/node_modules/**', 'node_modules/**']
-    }),
-    resolve({
-      jsnext: true,
-      browser: true,
-      preferBuiltins: true
-    }),
-    commonjs(),
-    globals(),
-    builtins(),
-    terser(),
-    license({
-      banner: `/*! Cisco Spark SDK v${version} */`
-    })
-  ]
-};
+export default function ({versionNumber}) {
```

Comment:
How is this method called? Seems like we would need to pass it somewhere from the build script through to here?

Reply:
gets called with rollup `npm run build:script -- --ver=${version}`